### DISCUSSION TITLE
Unable to see the dropdown values of widget in decouple cms

### DISCUSSION THREAD

**Discussion by saleemhaq** [OrchardCMS/OrchardCore#10590] [Q&A]
Unable to see the dropdown values of widget in decouple cms when click on add widget button:

![widgetarea](https://user-images.githubusercontent.com/5202291/139497466-ce18934e-f00a-4c95-b16e-1b49f1387b77.PNG)

**Comment by Skrypt** [comment]
Which version of Orchard Core are you using? Can you see any issues in the browser developer console? Can't repro in latest dev branch. These dropdowns are part of Bootstrap. So you probably have no widgets in your Content Types.

**Reply by saleemhaq** [reply]
No i not see any issue in console :( and i unable to see version of that pfa, please help
![adminarea](https://user-images.githubusercontent.com/5202291/139502641-45a34b33-36b3-4181-b99c-259c354902fa.png)
a.

**Reply by saleemhaq** [reply]
> Which version of Orchard Core are you using? Can you see any issues in the browser developer console? Can't repro in latest dev branch. These dropdowns are part of Bootstrap. So you probably have no widgets in your Content Types.

So what could i do for this to their is options come?

**Reply by Skrypt** [reply]
Take a screenshot of the Content Definitions list. If there is no Content Type set as a widget this is why it doesn't display any content item in the button dropdown.

**Reply by saleemhaq** [reply]
How it call in razor page ? in an specific area of page

**Reply by Skrypt** [reply]
In the admin UI, you go to Content --> Content Definitions

**Reply by saleemhaq** [reply]
Need to put custom input fields like first name, last name, contact and country dropdown and will save this data in db and need to show all that data in listing in admin side how I achieve this help me out.?

**Reply by Skrypt** [reply]
You can do this with Content Types. Please experiment with different things. You can use a "predefined list" with a TextField for countries. Of course Orchard Core has some learning curve. But it's no different than Drupal for example. For more global direct help, please ask on the Gitter forum.

**Reply by saleemhaq** [reply]
> You can do this with Content Types. Please experiment with different things. You can use a "predefined list" with a TextField for countries. Of course Orchard Core has some learning curve. But it's no different than Drupal for example. For more global direct help, please ask on the Gitter forum.

How i handle this event on button click and post all values on our custom controller and save this data in db  and later on show in admin side in an specific menu?

**Reply by saleemhaq** [reply]
Any one please help on above query?

**Comment by ns8482e** [comment]
Only content type with stereotype  `Widget` are available in Layers - Set your Stereotype of your Content type as Widget

**Reply by saleemhaq** [reply]
> Only content type with stereotype `Widget` are available in Layers - Set your Stereotype of your Content type as Widget

but How i handle this event on button click and post all values on our custom controller and save this data in db and later on show in admin side in an specific menu?

**Reply by ns8482e** [reply]
Using themes and display management  with `Forms` feature enabled - you create and place it in Layers.  The form has action url - you can place any url that handle submission - either it could be your custom controller or it could be workflow url.

https://docs.orchardcore.net/en/latest/docs/reference/modules/Forms/#forms-orchardcoreforms

**Reply by saleemhaq** [reply]
> Using themes and display management with `Forms` feature enabled - you create and place it in Layers. The form has action url - you can place any url that handle submission - either it could be your custom controller or it could be workflow url.
> 
> https://docs.orchardcore.net/en/latest/docs/reference/modules/Forms/#forms-orchardcoreforms
Please advise on belw:

![widgetIssue](https://user-images.githubusercontent.com/5202291/139588309-63b34cbd-2abd-400c-b522-e590a1ef6cbc.png)

**Reply by saleemhaq** [reply]
> > Using themes and display management with `Forms` feature enabled - you create and place it in Layers. The form has action url - you can place any url that handle submission - either it could be your custom controller or it could be workflow url.
> > https://docs.orchardcore.net/en/latest/docs/reference/modules/Forms/#forms-orchardcoreforms
> > Please advise on belw:
> 
> ![widgetIssue](https://user-images.githubusercontent.com/5202291/139588309-63b34cbd-2abd-400c-b522-e590a1ef6cbc.png)


The page could not find text come when calling widget.

**Reply by ns8482e** [reply]
@saleemhaq It may be due to not having route for `~/` - Add route for home page and it will render home page

**Reply by saleemhaq** [reply]
It calling on page it work but the text "The page could not be found" is also come?

**Reply by saleemhaq** [reply]
> It calling on page it work but the text "The page could not be found" is also come? when I call this widget.
